css/global.css

```diff
@@ -1,3 +1,7 @@
-img[src=*birthday] {
-  display: none;
+img[src*="birthday"] {
+  display: none !important;
+}
+
+iframe[name="account"] {
```

Comment:
Selects the iframe with the name attribute of "account" and removes it from the DOM. Since the birthday animation is in the iframe it's impossible to just select that and remove it from the DOM.